### PR TITLE
Overhauling Anything Minecraft

### PR DESCRIPTION
**Pull request by Silverarmor**
Please see commit descriptions for more information.

My notes (copied from Discord)

> A few of the notes I've taken upon my first read through:
> - What are you guys using to edit the md files? They don't appear linted, nor have a set format. I'd recommend a STYLE GUIDE.md for everyone to be on the same page.
> - There are a lot of people making multiple PRs, rather than putting multiple commits on a PR. This will lead to merge conflicts in the future - a big pain in the ass lol
> - What's the aim of this project? A CORE_VALUES.md/similar? I note that at the moment, the pages are quite subjective, and don't provide an overall/"fair" view/comparison between different server jars (as an example).
> - Tree/File placement. For a project to be successfull, we'd need to have something prepared for this. Currently it's hard to navigate between files, there's no directory or table of contents etc.

### CODE REVIEW

**Comment by Silverarmor** [comment]
I think we should use the AC spec
https://github.com/all-contributors/all-contributors

**Comment by Kruemmelspalter** [comment]
> There are a lot of people making multiple PRs, rather than putting multiple commits on a PR. This will lead to merge conflicts in the future - a big pain in the ass lol
I'm working on contributing guidelines (in branch guidelines)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributing Guidelines
```
Comment (Kruemmelspalter) [inline CONTRIBUTING.md:1]: This is already in #30 (with content)

File: README.md [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
-#### Setup
+### Setup
 
 - [Finding what server jar to use](./tutorials/en_us/Finding%20what%20server%20jar%20to%20use.md)
 - [Setting up your minecraft server](./tutorials/en_us/Setting%20up%20your%20minecraft%20server.md)
 
 ## Contributing
 
-If you would like to Contribute just make a pull request with the changes and I'll review it
+If you would like to contribute, pleae make a pull request.
```
Comment (Kruemmelspalter) [inline README.md:16]: Maybe add the link to the CONTRIBUTING.md and correct "pleae" to "please"

**Review by Kruemmelspalter — CHANGES_REQUESTED** [review]

**Comment by Silverarmor** [comment]
oopsies no need to assign me the PR, it's my PR haha

File: README.md (hunk 2 above)
Comment (Silverarmor) [reply README.md:16]: Just accept HEAD when resolving conflicts later.

**Review by JustDoom — APPROVED** [review]

**Review by Kruemmelspalter — APPROVED** [review]

**Pull request merged**